app/scripts/views/base.js

```diff
@@ -87,10 +87,14 @@ function (Cocktail, _, Backbone, Raven, $, p, AuthErrors,
     constructor: function (options) {
       options = options || {};
 
+      if (! options.translator) {
+        throw new Error('Translator is required');
+      }
```

Comment:
I have a branch I was looking at a few weeks ago that does this exact same thing. ;)